examples/planewave-quad-comparison.py

```diff
@@ -0,0 +1,197 @@
+"""
+Tensor Product Quadrature vs. Vioreanu-Rokhlin Quadrature for Plane Wave on Sphere
+==================================================================================
+
+This test compares the absolute error of **Tensor Product Quadrature** and
+**Vioreanu-Rokhlin Quadrature** against the number of discretization nodes
+with matched total polynomial degree exactness.
+
+Comparison of Polynomial Exactness
+----------------------------------
+
+The following table summarizes the total degree of polynomial exactness of both
+quadrature methods based on the order:
+
+Order     VR Exact_to     Tensor Exact_to
+-----------------------------------------
+1         2              3
+2         4              5
+3         5              7
+4         7              9
+5         8              11
+6         10             13
+7         12             15
+8         14             17
+9         15             19
+10        17             21
+11        19             23
+12        20             25
+13        22             27
+14        24             29
+15        25             31
+16        27             33
+17        28             35
+18        30             37
+19        32             39
```

Comment:
It's not useful to incorporate this table, since it's not guaranteed to match the code on an ongoing basis. Instead, the code should generate the table.